lib/setupmanagers/qemuhck/network_manager.rb

```diff
@@ -49,8 +49,8 @@ def read_world_ip(device_name, qemu_replacement_list = {})
         replacement_list = device_replacement_list('world', device, type_config, qemu_replacement_list)
         mac = replace_string_recursive(type_config['mac'], replacement_list)
 
-        line = File.readlines('/proc/net/arp')[1..].map(&:split).find do
-          _1[3] == mac && _1[5] == 'br_world'
+        line = File.readlines('/proc/net/arp')[1..].map(&:split).find do |arp_line|
```

Comment:
Distinguishing two variables by naming `line` and `arp_line` sounds too ambiguous. Perhaps we may use rename `line` to `found` and `arp_line` to `candidate`. You may not prefer `candidate` since it is a bit verbose though.

Reply:
Moved to https://github.com/HCK-CI/AutoHCK/pull/164